### PR TITLE
fix the queue creation endpoint

### PR DESCRIPTION
**Pull request by zwang180**
#84 , seems to be the best thing I can do, given scope does not apply to create or build.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ with the current date and the next changes should go under a **[Next]** header.
 
 ## [Next]
 
+* Correctly display number of questions when first creating a queue.
```
Comment (nwalters512) [inline CHANGELOG.md]: You can keep everything on one line :)

File: api/queues.js [hunk 2]
```diff
@@ -49,19 +49,23 @@ router.post(
...
       name: data.name,
       location: data.location,
       fixedLocation: data.fixedLocation === true,
       courseId,
       createdByUserId: res.locals.userAuthn.id,
     })
 
-    queue.save().then(newQueue => res.status(201).json(newQueue))
+    Queue.scope('questionCount')
+      .findOne({
```
Comment (nwalters512) [inline api/queues.js]: You can use `findById(1234)` to make this a little more concise.

File: CHANGELOG.md (hunk 1 above)
Comment (zwang180) [reply CHANGELOG.md]: lol, forget it is a markdown :)

File: api/queues.js (hunk 2 above)
Comment (zwang180) [reply api/queues.js]: Done, please check

**Pull request merged**